Library/Homebrew/utils/popen.rb

```diff
@@ -12,6 +12,8 @@ def self.popen_write(*args, &block)
   end
 
   def self.popen(args, mode)
+    args = args.map { |a| a.respond_to?(:to_path) ? a.to_path : a }
```

Comment:
Are you sure about calling `#to_path` on _all_ of the arguments? It's only the first argument that should always be an object representing a path, IMO — generic shell arguments are just strings with no special meaning, so I wonder if those should just be converted straight to strings. Very minor point, I know, but I'm a bit of a conversion protocol enthusiast so this kind of thought appeals to me. :P